boxoffice/templates/boxoffice.js

```diff
@@ -888,5 +888,17 @@ $(function() {
   window.Boxoffice.init = function(widgetConfig) {
 	  boxoffice.init(widgetConfig);
   };
+
+  // Raise a custom event once boxoffice.js has been loaded
+  if (typeof(window.Event) === "function") {
+    var event = new Event('boxofficeScript-load');
+  }
```

Comment:
Can you please collapse these two lines into a single line, since we're using the following convention:
```
if (condition) {
 //
} else {
  //
}
```

Reply:
Fixed